packages/service-library/src/servicelib/utils.py

```diff
@@ -61,17 +61,17 @@ def search_osparc_repo_dir(start: Union[str, Path], max_iterations=8) -> Optiona
 
 
 # FUTURES
+def log_exception_callback(fut: asyncio.Future):
```

Comment:
can you make this private if it is used only in this module?

Reply:
it is also used abroad (see ``projects_api.delete_project``), that is why i moved it outside of ``fire_and_forget``